src/preferences/dialog/dlgprefsounddlg.ui

```diff
@@ -329,6 +329,16 @@
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0" colspan="2">
+       <widget class="QLabel" name="deckBusHint">
+        <property name="text">
+         <string>Deck and Bus outputs are for external mixers. They are post-fader and include effects and crossfader (for Auto DJ). For manual mixing, set all Mixxx faders to unity (open). </string>
```

Comment:
I doubt the wording "faders at unity" is clear for all users, especially for non-techy beginners.
Let's use someting that refers to the GUI:
```suggestion
         <string>Deck and Bus outputs are for external mixers. They are post-fader and include effects and crossfader (for Auto DJ). For external mixing, make sure all Mixxx faders and EQ knobs are set to their default position (right- or double-click). </string>
```